spyglass/src/main/java/com/linkedin/android/spyglass/ui/MentionsEditText.java

```diff
@@ -759,7 +759,7 @@ private void removeTextWithinDeleteSpans(@NonNull Editable text) {
      * @param text the final version of the text after it was changed
      */
     private void replacePlaceholdersWithCorrespondingMentionSpans(@NonNull Editable text) {
-        PlaceholderSpan[] tempSpans = text.getSpans(0, text.length(), PlaceholderSpan.class);
+        PlaceholderSpan[] tempSpans = text.getSpans(0, text.length()-1, PlaceholderSpan.class);
```

Comment:
Two things:

1. It's not clear to me from the description what the problem is. Providing more details (especially steps to reproduce the issue) would be useful.

2. This code is just trying to get all PlaceholderSpans from the text. I'm not sure why we are ignoring the last character in the text (which is what this code would be doing from what I can tell).